docs/src/integrations/exchange.md

```diff
@@ -567,6 +566,19 @@ public class PubkeyValidator
 }
 ```
 
+## Minimum Deposit & Withdrawal Amounts
+
+Every deposit and withdrawal must be greater or equal to the minimum rent-exempt
+balance for a 0-data account, currently: 0.000890880 SOL
```

Comment:
nit: `a 0-data account (i.e the account signified as normal wallet address)` (sad, we don't have a proper word. i borrowed `wallet` from the ata parameter names).

Reply:
Yep, good clarification. Hopefully you like the rewrite I did. Otherwise, happy to iterate more post-merge.